src/rules/function-url-path/__tests__/index.js

```diff
@@ -0,0 +1,163 @@
+    description: "ignore variable",
+  }, {
+    code: "a { background: url(var(--image)); }",
+    description: "ignore variable",
+  } ],
+
+  reject: [ {
+    code: "a { background: url(./); }",
+    message: messages.rejected,
+    line: 1,
+    column: 5,
+  }, {
+    code: "a { background: url(http.jpg); }",
+    message: messages.rejected,
+    line: 1,
+    column: 5,
+  }, {
+    code: "a { background: url(file.jpg); }",
+    message: messages.rejected,
+    line: 1,
+    column: 5,
+  }, {
+    code: "a { background: url(./file.jpg); }",
+    message: messages.rejected,
+    line: 1,
+    column: 5,
+  }, {
+    code: "a { background: url(../file.jpg); }",
+    message: messages.rejected,
+    line: 1,
+    column: 5,
+  }, {
+    code: "a { background: URL(../file.jpg); }",
+    message: messages.rejected,
+    line: 1,
+    column: 5,
+  }, {
+    code: "a { background: url('../file.jpg'); }",
+    message: messages.rejected,
+    line: 1,
+    column: 5,
+  }, {
+    code: "a { background: url(\"../file.jpg\"); }",
+    message: messages.rejected,
+    line: 1,
+    column: 5,
+  }, {
+    code: "@font-face { font-family: 'foo'; src: url('./file.jpg'); }",
+    message: messages.rejected,
+    line: 1,
+    column: 34,
+  } ],
+})
+
+testRule(rule, {
+  ruleName,
+  config: ["relative"],
+  skipBasicChecks: true,
+
+  accept: [ {
+    code: "@import url();",
+  }, {
+    code: "@import url('');",
+  }, {
+    code: "@import url(\"\");",
+  }, {
+    code: "a { background: url(file.jpg); }",
+  }, {
+    code: "a { background-image: url('data:image/gif;base64,R0lGODlhAQABAIAAAAUEBAAAACwAAAAAAQABAAACAkQBADs='); }",
+  }, {
+    code: "a { background: url(./file.jpg); }",
+  }, {
+    code: "a { background: url(../file.jpg); }",
+  }, {
+    code: "a { background: URL(../file.jpg); }",
+  }, {
+    code: "a { background: url('../file.jpg'); }",
+  }, {
+    code: "a { background: url(\"../file.jpg\"); }",
+  }, {
+    code: "@font-face { font-family: 'foo'; src: url('./path/to/foo.ttf'); }",
+  }, {
+    code: "a { background: some-url(); }",
+    description: "ignore contain url function",
+  }, {
+    code: "a { background: url($image); }",
+    description: "ignore variable",
+  }, {
+    code: "a { background: url(@image); }",
+    description: "ignore variable",
+  }, {
+    code: "a { background: url(var(--image)); }",
+    description: "ignore variable",
+  } ],
+
+  reject: [ {
+    code: "a { background: url(/); }",
+    message: messages.rejected,
+    line: 1,
+    column: 5,
+  }, {
+    code: "a { background: URL(/file.jpg); }",
+    message: messages.rejected,
+    line: 1,
+    column: 5,
+  }, {
+    code: "a { background: url('/file.jpg'); }",
+    message: messages.rejected,
+    line: 1,
+    column: 5,
+  }, {
+    code: "a { background: url(\"/file.jpg\"); }",
+    message: messages.rejected,
+    line: 1,
+    column: 5,
+  }, {
+    code: "a { background: url('//www.google.com/file.jpg'); }",
```

Comment:
@ntwb for disable use this anti-pattern we have rule https://github.com/stylelint/stylelint/pull/1576